src/main/java/com/rultor/agents/github/CommentsTag.java

```diff
@@ -109,6 +122,20 @@ public CommentsTag(final Github ghub) {
                 )
             );
             Logger.info(this, "duplicate tag %s commented", tag);
+        } else if (CommentsTag.valid(tag)
+            && !CommentsTag.valid(tag, previous)) {
```

Comment:
@gumbelmj If `CommentsTag.valid(tag)` returns false (i.e: the release tag does not comply with the regex), the code will fall to the `else` branch and rultor will try to release anyway.